app/uk/gov/nationalarchives/omega/editorial/connectors/ApiConnector.scala

```diff
@@ -85,6 +97,19 @@ class ApiConnector @Inject() (
 
 object ApiConnector {
 
+  sealed abstract class SID(val value: String) {
+
+    def matches(sid: String): Boolean =
+      sid.trim.equalsIgnoreCase(this.value)
+
+  }
+
+  object SID {
+    case object GetEditSet extends SID("OSGEES001")
+    case object GetEditSetRecord extends SID("OSGESR001")
+    case object UpdateEditSetRecord extends SID("OSUESR001")
+  }
```

Comment:
You could use an [enumeratum](https://github.com/lloydmeta/enumeratum) for this.